### PR TITLE
Added - i18n configuration

### PR DESCRIPTION
**Pull request by MatiasComercio**
# Summary
- All blog webpage is now both in Spanish and English.
- Locales were set on URL to match API RESTful expected behavior.
- Locales can be switched using a drop-down menu, accesible over all the blog.
- Bootstrap has been added because it was easier to work with its lists rather than crude html lists (\<select\> \<option\> tags) due to link manipulation to change among locales.

## Remains
- Errors haven't been translated because I did not know how to do it. Help
is needed.
- Spanish's pluralization are not working.

# Trello Card
https://trello.com/c/kqbMEQLb/10-internacionalizar-el-blog

# Snapshots
## English Index
english

## Drop-down menu
english_spanish

## Spanish Index
spanish

### CODE REVIEW

**Comment by epintos** [comment]
@MatiasComercioITBA I can see there are lots of commits from other PR, what happened?

**Comment by MatiasComercioITBA** [comment]
I have no idea. I've merged changes with master branch and this was the result. Perhaps I should have rebased it, not merged it. I'll fix it immediately.

File: config/locales/en.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+en:
+  index_title: Welcome Index
```
Comment (epintos) [inline config/locales/en.yml]: Its better to have hierarchy here, for example:

```
articles:
   title: Title
   create:
      success: Algo
       error: algo
```
And then you do `t('articles.create.success')`

File: config/routes.rb [hunk 2]
```diff
@@ -1,12 +1,23 @@
 Rails.application.routes.draw do
 
-	resources :articles do
-		resources :comments
-	end
+  # For i18n urls, like http://www.example.org/en/
+  # "I18n.available_locales.join("|")" taken from:
+  # http://stackoverflow.com/questions/8224245/rails-routes-with-optional-scope-locale
+  scope '(:locale)', locale: /#{I18n.available_locales.join("|")}/ do
```
Comment (epintos) [inline config/routes.rb]: Its better to manage the I18n with query param `?locale` instead of adding it in the url. This is to avoid having dirty urls

**Comment by epintos** [comment]
👍

**Pull request merged**